packages/browser/src/index.test.ts

```diff
@@ -71,9 +109,9 @@ describe('LogtoClient', () => {
         requester
       );
 
-      // @ts-expect-error expected to set undefined
+      // @ts-expect-error expected to set null
       signInSessionAccessor.setSignInSessionItem();
```

Comment:
should pass in null instead of disable the rule

Reply:
Fixed. PTAL.